install/playbooks/main-inline.yml

```diff
@@ -7,72 +7,113 @@
     - '{{ playbook_dir }}/../../config/defaults.yml'
   roles:
     - role: load-defaults
+      tags: always
     - role: external-ip-type
+      tags: always
     - role: packages
+      tags: apt
     - role: system-prepare
+      tags: system
     - role: remote-access
+      tags: ssh
     - role: luks-remote
+      tags: luks
     - role: nginx
+      tags: nginx
     - role: dns-server-bind
       when: bind.install
+      tags: bind
     - role: dns-server-check-propagation
       when: bind.install and bind.propagation.check
+      tags: bind
     - role: certificates-all
+      tags: ssl
     - role: ldap
+      tags: ldap
     - role: user-setup
+      tags: user
     - role: access-check
       when: access_check.active
+      tags: access
     - role: access-report
       when: access_check.active
+      tags: access
     - role: backup-server
+      tags: borg
     - role: rspamd
       when: mail.antispam.active
+      tags: rspamd
     - role: rspamd-web
       when: mail.antispam.webui.active
+      tags: rspamd,nginx
     - role: clamav
       when: mail.antivirus.active
+      tags: clamav
     - role: opendkim
+      tags: dkim
     - role: opendmarc
+      tags: dmarc
     - role: postfix
+      tags: postfix
     - role: mta-sts
+      tags: mta-sts
     - role: dovecot
+      tags: dovecot
     - role: roundcube
       when: webmail.install and webmail.type == 'roundcube'
+      tags: roundcube
     - role: roundcube-milters
       when: webmail.install and webmail.milters.active
+      tags: roundcube
     - role: sogo
       when: sogo.install
+      tags: sogo
     - role: sogo-milters
       when: sogo.install and sogo.milters.active
+      tags: sogo
     - role: ejabberd
       when: ejabberd.install
+      tags: jabber
     - role: website-simple
       when: website.install
+      tags: nginx,www
     - role: autodiscover
       when: mail.autodiscover
+      tags: autodiscover
     - role: autoconfig
       when: mail.autoconfig
+      tags: autoconfig
     - role: borg-backup
       when: backup.install
+      tags: borg
     - role: transmission
       when: transmission.install
+      tags: transmission
     - role: dns-server-bind-refresh
       when: bind.install
+      tags: bind
     - role: tor
       when: tor.install
+      tags: tor
     - role: privoxy
       when: privoxy.install
+      tags: privoxy
     - role: zabbix-server
       when: zabbix.install
+      tags: zabbix
     - role: import-accounts
       when: mail.import.active and not system.devel
+      tags: import
     - role: extra-certs
       when: extra_certs is defined
+      tags: ssl
     - role: well-known-services
+      tags: nginx
     - role: system-post-install
+      tags: always
```

Comment:
I don't think the `system-post-install` needs to be run `always`.
It will disable the root access via SSH for example: not really expected if running a single tagged aspect.

Reply:
In this case, I would prefer to fix the role, but having a role at the end will be useful. For instance to clean-up package cache, lock some partitions in r/o mode, etc...